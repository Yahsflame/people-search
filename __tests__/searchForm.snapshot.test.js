import React from 'react';
import form from '../components/SearchForm';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<form page="https://people-search-app.surge.sh">Versium</form>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});