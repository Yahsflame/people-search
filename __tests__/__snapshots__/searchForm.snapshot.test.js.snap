// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  page="https://people-search-app.surge.sh"
>
  Versium
</form>
`;
